src/content/blog/2025_quantum_device_workshop.md

```diff
@@ -0,0 +1,139 @@
+
+- **Content and Switching Tracks** -- I think we allowed people to switch between tracks even if we encouraged people to just stick to one. During the workshop we encouraged people who thought the advanced track talks and workshops were too hard to switch to the beginner track. I think next year we need to have a syllabus with pre-requisites for each track. 
+
+- **Hybrid Quantum Devices** -- Potential talk on cQED for coupling hybrid quantum devices. There is an increasing interest in other quantum hardware communities in using superconductors to couple devices together. 
+
+- **More qubit types** -- Some people wanted to learn more about Fluxonium or Bosonic qubits. There is indeed a very pedagogical way to include this into the workshop. For Fluxonium, we can extend Michel Devoret's talk to talk more about flux versus charge coupling and control in cQED and build from talks about phase and flux qubits versus charge qubits and tunable transmons. In addition, designing Josephson Junction arrays in a way that avoids fabrication defects could be interesting to talk about. For Bosonic qubits, talking about how to design high Q resonators or cavities versus the design of low Q readout resonators could be an interesting topic and would allow us to introduce important considerations such as internal and external Q. 
+
+- **More on Amplifiers and other circuit components** -- Purcell filters (and Purcell filter-like components) and amplifiers are becoming an increasingly important part of scaling superconducting qubit systems and we maybe could spend more time talking about them. Many parametric amplifiers are useful for coupling between bosonic qubits (eg. SNAILs and 3 wave mixing), nonlinear effects, or squeezing experiments. 
+
+
+## Final Thoughts 💭
+
+Overall, it was a successful event, and we will do it again next year! We were excited to see all the enthusiasm from the community towards designing scalable quantum devices and hope people continue to be very enthusiastic to design more devices and learn more. We also hope that this workshop will bring more people to help maintain open source quantum hardware design tools and create more accessible educational content! The Quantum Computing Student Association, the event sponsors, speakers, and advisory board worked hard to make this event possible and we thank everyone involved to make this event amazing. Please join the [Discord](https://discord.gg/9yYmbDPH) for open source quantum hardware design tools and accessible quantum hardware education if you haven't already. We have already started to organize next year's iteration for the quantum device workshop, and the link to pre-register is [here](https://qdw-ucla.squarespace.com/qdw2026)!
```

Comment:
@auratrifire the Discord link is invalid/expired. Can you provide us with a new link without expiration?